@kiva/kv-components/vue/KvCommentsReplyButton.vue

```diff
@@ -0,0 +1,43 @@
+<!-- eslint-disable max-len -->
+<template>
+	<button
```

Comment:
In this repo we tend to add simple unit tests for new components. This component should get a test to validate the event is emitted.